src/ucp/proto/proto_init.h

```diff
@@ -68,44 +49,40 @@ ucp_proto_perf_envelope_make(const ucs_linear_func_t *funcs, uint64_t funcs_num,
                              ucp_proto_perf_envelope_t *envelope_list);
 
 
-/**
- * Initialize the performance of a protocol that consists of several parallel
- * stages. The performance estimations are added to params->caps.
- *
- * @param [in]    proto_name    Protocol name, for debugging.
- * @param [in]    range_start   Range interval start.
- * @param [in]    range_end     Range interval end.
- * @param [in]    bias          Performance bias (0 - no bias).
- * @param [in]    stages        Array of parallel stages performance ranges.
- * @param [in]    num_stages    Number of parallel stages in the protocol.
- * @param [inout] caps          Filled with protocol performance data.
- */
+ucp_proto_common_init_params_t
+ucp_proto_common_params_init(const ucp_proto_init_params_t *init_params);
```

Comment:
Maybe `ucp_proto_common_init_params`, and move it to `ucp_proto_common.h`

Reply:
Done